src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -361,7 +337,7 @@ def work_for_vts_rsc(vts, ctx):
         classpath_product = self.context.products.get_data('rsc_mixed_compile_classpath')
         classpath_entries = classpath_product.get_classpath_entries_for_targets(dependencies_for_target)
         for _conf, classpath_entry in classpath_entries:
-          classpath_paths.append(classpath_entry.path)
+          classpath_paths.append(fast_relpath(classpath_entry.path, get_buildroot()))
```

Comment:
Keeping the path as absolute within the `ClasspathEntry` itself seems definitely like the right thing to do here -- thanks for making this consistent compared to the first diff I had posted!